### PR TITLE
update neo mxnet notebooks for v1.8

### PR DESCRIPTION
**Pull request by prats13bag**
**Description of changes:**

With this PR following four example notebooks and associated entry_point scripts are being updated to use the latest supported version i.e. 1.8
- [imageclassification_caltech](https://github.com/aws/amazon-sagemaker-examples/tree/master/sagemaker_neo_compilation_jobs/imageclassification_caltech)
- [gluoncv_ssd_mobilenet](https://github.com/aws/amazon-sagemaker-examples/tree/master/sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet)
- [gluoncv_yolo_darknet](https://github.com/aws/amazon-sagemaker-examples/tree/master/sagemaker_neo_compilation_jobs/gluoncv_yolo_darknet)
- [mxnet_mnist](https://github.com/aws/amazon-sagemaker-examples/tree/master/sagemaker_neo_compilation_jobs/mxnet_mnist)

**Testing:**

All the four notebooks runs successfully to get correct prediction results.

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: sagemaker_neo_compilation_jobs/mxnet_mnist/input.html [hunk 1]
```diff
@@ -1,270 +0,0 @@
-<script type="text/Javascript">
```
Comment (samskalicky) [inline sagemaker_neo_compilation_jobs/mxnet_mnist/input.html:1]: should we consider leaving this here, and adding a separate python script called "get_input.py" that does what the notebook used to do? And add something that saves the data as *.npy file? Then if people wanna go generate new inputs they can... just a thought.

File: sagemaker_neo_compilation_jobs/mxnet_mnist/mxnet_mnist_neo.ipynb [hunk 2]
```diff
@@ -212,9 +208,7 @@
     "\n",
     "Now that our Endpoint is deployed and we have a ``predictor`` object, we can use it to classify handwritten digits.\n",
     "\n",
-    "To see inference in action, draw a digit in the image box below. The pixel data from your drawing will be loaded into a ``data`` variable in this notebook. \n",
-    "\n",
-    "**Note:** After drawing the image, you'll need to move to the next notebook cell."
+    "To see inference in action, we define a variable `data` and initialize it with the binary equivalent of a hand drawn digit `0`"
```
Comment (samskalicky) [inline sagemaker_neo_compilation_jobs/mxnet_mnist/mxnet_mnist_neo.ipynb:211]: we can refer to this other python script here, saying how we got this input data

**Review by samskalicky — APPROVED** [review]
minor comments, but overall LGTM. Thanks @prats13bag

File: sagemaker_neo_compilation_jobs/mxnet_mnist/input.html (hunk 1 above)
Comment (prats13bag) [reply sagemaker_neo_compilation_jobs/mxnet_mnist/input.html:1]: Thanks for taking this up offline. We are now good to merge it.

File: sagemaker_neo_compilation_jobs/mxnet_mnist/mxnet_mnist_neo.ipynb (hunk 2 above)
Comment (prats13bag) [reply sagemaker_neo_compilation_jobs/mxnet_mnist/mxnet_mnist_neo.ipynb:211]: Thanks for taking this up offline. We are now good to merge it.

**Review by ngluna — DISMISSED** [review]

**Review by samskalicky — APPROVED** [review]

**Pull request merged**